### ISSUE TITLE
lower KITTI 2D Tracking inference resullt 

### ISSUE THREAD

**Issue by yustaub** [xingyizhou/CenterTrack#162]
nice work firstly. I download the pre-trained model and create the kitti data info and inference the tracking, and I get the lower MOTA than yours, and you also said that `We observe up to 1.5 MOTA jittering due to randomness` is it because the centernet Gaussion distribution? Very appreciate for you reply!

**Comment by lushbng** [comment]
me too . but test.py has a error no such file or directory: '. / tools / Eval_ kitti_ track/data/tracking/evaluate_ trackingtest.seqmap '.

**Comment by yustaub** [comment]
evaluate_ trackingtest.seqmap is used when you want to run centerTrack on kitti test set. you can replace it to 'evaluate_ trackingval.seqmap' in code @lushbng

**Comment by lushbng** [comment]
I've solved this problem. Thank you. Mota has 87.



At 2020-12-18 17:39:09, "yustaub" <notifications@github.com> wrote:

evaluate_ trackingtest.seqmap is used when you want to run centerTrack on kitti test set. you can replace it to 'evaluate_ trackingval.seqmap' in code @lushbng
